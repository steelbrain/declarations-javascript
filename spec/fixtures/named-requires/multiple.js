const { x: xx } = require('file-x')
const { y: yy } = require('file-y')
const { z } = require('file-z')
xx()
yy()
