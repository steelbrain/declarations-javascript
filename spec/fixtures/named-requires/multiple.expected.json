[
  {
    "name": null,
    "position": {
      "start": {
        "line": 1,
        "column": 26
      },
      "end": {
        "line": 1,
        "column": 34
      }
    },
    "source": {
      "name": null,
      "filePath": "file-x",
      "position": {
        "start": {
          "line": 1,
          "column": 26
        },
        "end": {
          "line": 1,
          "column": 34
        }
      }
    }
  },
  {
    "name": null,
    "position": {
      "start": {
        "line": 2,
        "column": 26
      },
      "end": {
        "line": 2,
        "column": 34
      }
    },
    "source": {
      "name": null,
      "filePath": "file-y",
      "position": {
        "start": {
          "line": 2,
          "column": 26
        },
        "end": {
          "line": 2,
          "column": 34
        }
      }
    }
  },
  {
    "name": null,
    "position": {
      "start": {
        "line": 3,
        "column": 22
      },
      "end": {
        "line": 3,
        "column": 30
      }
    },
    "source": {
      "name": null,
      "filePath": "file-z",
      "position": {
        "start": {
          "line": 3,
          "column": 22
        },
        "end": {
          "line": 3,
          "column": 30
        }
      }
    }
  },
  {
    "name": "xx",
    "position": {
      "start": {
        "line": 4,
        "column": 0
      },
      "end": {
        "line": 4,
        "column": 2
      }
    },
    "source": {
      "name": null,
      "filePath": null,
      "position": {
        "start": {
          "line": 1,
          "column": 11
        },
        "end": {
          "line": 1,
          "column": 13
        }
      }
    }
  },
  {
    "name": "yy",
    "position": {
      "start": {
        "line": 5,
        "column": 0
      },
      "end": {
        "line": 5,
        "column": 2
      }
    },
    "source": {
      "name": null,
      "filePath": null,
      "position": {
        "start": {
          "line": 2,
          "column": 11
        },
        "end": {
          "line": 2,
          "column": 13
        }
      }
    }
  }
]